loader_hub/boarddocs/README.md

```diff
@@ -0,0 +1,21 @@
+# BoardDocs Loader
+
+This loader retrieves an agenda and associated material from a BoardDocs site.
+
+This loader is not endorsed by, developed by, supported by, or in any way formally affiliated with Diligent Corporation.
+
+## Usage
+
+To use this loader, you'll need to specify which BoardDocs site you want to load.
+
+```python
+from llama_index import download_loader
+
+BoardDocsReader = download_loader("BoardDocsReader")
+loader = BoardDocsReader(site="ca/redwood", committee_id="A4EP6J588C05")
+# For a site URL https://go.boarddocs.com/ca/redwood/Board.nsf/Public
+# your boarddocssite should be set to 'ca/redwood'
+documents = loader.load_data(boarddocssite=['ca/redwood'])
```

Comment:
is boarddocssite specified in the code? also why does `ca/redwood` need to be specified twice?

Reply:
Whoops, sorry - I changed the calling patterns but hadn't updated the README. Fixed.